### PR TITLE
Stop mapping errata packages at build_done

### PR DESCRIPTION
**Pull request by javihernandez**
Resolves: https://github.com/AlmaLinux/build-system/issues/402

### CODE REVIEW

File: alws/crud/build_node.py
Comment (pastalian) [inline alws/crud/build_node.py:1]: Some of the imports no longer seem to be necessary after this change.
```
alws/crud/build_node.py:17:0: W0611: Unused ErrataPackageStatus imported from alws.constants (unused-import)
alws/crud/build_node.py:17:0: W0611: Unused GitHubIssueStatus imported from alws.constants (unused-import)
alws/crud/build_node.py:31:0: W0611: Unused move_issue_to_testing imported from alws.utils.github_integration_helper (unused-import)
alws/crud/build_node.py:38:0: W0611: Unused clean_release imported from alws.utils.parsing (unused-import)
alws/crud/build_node.py:38:0: W0611: Unused parse_rpm_nevra imported from alws.utils.parsing (unused-import)
```

Comment (javihernandez) [reply alws/crud/build_node.py:1]: nice catch, fixed now

File: alws/crud/build_node.py [hunk 1]
```diff
@@ -442,72 +421,6 @@ def append_errata_package(_, errata_package, artifact, rpm_info):
...
-                continue
-            errata_package.source_srpm = src_name
-            await db.run_sync(
-                append_errata_package,
-                errata_package,
-                build_task_artifact,
-                rpm_info,
-            )
-            errata_record_ids.add(errata_package.errata_record_id)
-    if settings.github_integration_enabled and errata_record_ids:
```
[the remark this answers is not in the document]
Comment (javihernandez) [reply alws/crud/build_node.py:499]: Fair point, I guess that I wanted to get rid of the kind of flooding we're getting in [updates issues like this one](https://github.com/AlmaLinux/updates/issues/958). 
@andrewlukoshko and @eabdullin1 what do you think?

[the remark this answers is not in the document]
Comment (javihernandez) [reply alws/crud/build_node.py:499]: Oh, sorry, their are not a side effect of build done but due to the initial errata package mapping upon errata insertion.
This one is about moving the status of the issue to testing, so yeah, I'm keeping it

[the remark this answers is not in the document]
Comment (javihernandez) [reply alws/crud/build_node.py:499]: done

**Review by anfimovdm — APPROVED** [review]

**Pull request merged**